pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -184,13 +184,27 @@ func canIgnoreHealthErrStatusInReconcile(cephCluster cephv1.CephCluster, control
 		}
 	}
 
-	// If there is only one cause for HEALTH_ERR and it's on the allowed list of errors, ignore it.
-	var allowedErrStatus = []string{"MDS_ALL_DOWN"}
-	var ignoreHealthErr = len(healthErrKeys) == 1 && contains(allowedErrStatus, healthErrKeys[0])
-	if ignoreHealthErr {
-		logger.Debugf("%q: ignoring ceph status %q because only cause is %q (full status is %+v)", controllerName, cephCluster.Status.CephStatus.Health, healthErrKeys[0], cephCluster.Status.CephStatus)
+	// If there are no keys causing the HEALTH_ERR status, we can ignore it
+	if len(healthErrKeys) == 0 {
+		return false
```

Comment:
shouldn't this return as `true` instead of `false` ?

Reply:
It is intentionally returning false when there are 0. This was the result previous to my changes, and I found if I returned true it broken a few unit tests. For production code I don't think it is important which one to return though. I'll update the comment for clarity on it.